### ISSUE TITLE
Search input - Loading - Fix disappearing focus

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1233]
CENG_3312

Docs: https://react-magma.cengage.com/version/4.3.0/api/search/

Success Criteria: https://www.w3.org/WAI/WCAG21/Understanding/focus-order

For the Loading example, when the search button is focused on and then pressed the button is then hidden and focus is moved to the body. When this happens it could confuse a user of assistive technology.

Perkins recommendation: If the search button is removed, focus should be programmatically moved somewhere logical or the button could be left where it is while the loading happens.

Solution: Let's just stop hiding the button and keep focus on it.

**Comment by orion-cengage** [comment]
Tested this on the dev docs site. The Search button correctly shows the loading animation upon clicking, and then the focus remains on it even after it is finished. You never lose site of focus. This appears good to go for me.
